### ISSUE TITLE
2.4.1 package missing for M1 Macs

### ISSUE THREAD

**Issue by quulah** [UpCloudLtd/terraform-provider-upcloud#265]
```
Provider registry.terraform.io/upcloudltd/upcloud v2.4.1 does not have a package available for your current platform, darwin_arm64.
```

2.4.0 still had this, but for some reason it cannot be found for 2.4.1.

**Comment by Darep** [comment]
Moi Miika! 😄 Nice catch, we'll have a look 👀

**Comment by Darep** [comment]
I think we found the issue: we updated to Go 1.17 and the goreleaser version we are using has a bug regarding this 😄 https://github.com/goreleaser/goreleaser/issues/2412

**Comment by Darep** [comment]
We have now released v2.4.2 which should have the darwin-arm64 binary for M1 macs 👍

**Comment by quulah** [comment]
Tervehdys! :)

Yup, seems to be working now. Thanks!
